mteb/tasks/Clustering/zho/CMTEBClustering.py

```diff
@@ -1,11 +1,124 @@
 from __future__ import annotations
 
+import itertools
+
+from datasets import Dataset, DatasetDict
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
-from ....abstasks.AbsTaskClustering import AbsTaskClustering
+NUM_SAMPLES = 2048
```

Comment:
How well does the score compare? I could imagine that we might need more than 2048 (but I might be wrong).

Reply:
Will run the full thing as well to compare.

Reply:
- CLS P2P Fast: "evaluation_time": 5.24, "v_measure": 0.38429563913648135,
- CLS P2P: "evaluation_time": 156.89, "v_measure": 0.3919956383426169,